image-index.md

```diff
@@ -87,7 +87,11 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`features`** *array of strings*
 
-        This property is RESERVED for future versions of the specification.
+        This OPTIONAL property specifies an array of strings, each specifying a mandatory hardware/host feature. Examples are:
```

Comment:
"mandatory" doesn't really have meaning in this spec, you need to explain what the behaviour needs to be and how a runtime should check the features.